content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+... short discussion about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app"), 
+... link to kubeflow-overview for more details
+... (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+
+... short discussion about how Kubeflow is versioned, 
+... that is, components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+
+<a id="roll-your-own"></a>
+## Path 1: roll your own
```

Comment:
I would recommend removing the  "Path #:" prefix from the headings. Also, do not use slang such as "roll your own" to describe creating a distribution. Colloquialisms and slang can make documentation more difficult for non-english speaking readers to understand. Here are some suggestions for the headings:

- Create a custom distribution
- Create your own distribution
- Customize a distribution
- Select a distribution 
- Choose a distribution
- Use a distribution

Reply:
I have made this change, and replaced the names with:
* `Packaged distributions`
* `Directly use the manifests`